app/controllers/bus_stops_controller.rb

```diff
@@ -61,13 +61,12 @@ def name_search
 
   def outdated
     @date = Date.parse(params[:date]) rescue 1.month.ago.to_date
```

Comment:
I know I wrote the rescue, but it still bothers me that rescuing is the most effective way we have of protecting against silly dates.

Reply:
I can fix it in a separate PR if you want. I think it's outside the scope of this issue.

Reply:
Well, I don't think there's much to fix. Unless we have a new preferred method of handling this.

I just read it and it made me a little bit sad.

Reply:
I understand. It is sad.